conda-build-all

```diff
@@ -95,14 +95,14 @@ def main():
         '--python',
         help="Set the Python version used by conda build",
         metavar="PYTHON_VER",
-        default='27,35,36',
+        default='27,35,36,37',
 
     )
     p.add_argument(
         '--numpy',
         help="Set the NumPy version used by conda build",
         metavar="NUMPY_VER",
-        default='1.10,1.11,1.12',
+        default='1.13,1.14,1.15',
```

Comment:
@Lnaden : It looks like conda-forge is actively building numpy 1.9, 1.11, and 1.15 for Python 3.7 now:
https://anaconda.org/conda-forge/numpy/files

They're have also built numpy 1.10, 1.12, 1.13, and 1.14 for Python 2.7, 3.5, and 3.6.

This is a pretty weird combination.